docs/source/dimred/run.rst

```diff
@@ -263,12 +264,6 @@ Tutorial
     Your output should look similar to this:
 
     ::
-
-
-        ==== LASSO - AI ====

```

Comment:
Good catch :+1:  I Would leave the heading. What do you think about `OPEN-LASSO` instead?